src/main/java/edu/cornell/kfs/concur/batch/businessobject/ConcurRequestedCashAdvance.java

```diff
@@ -0,0 +1,86 @@
+package edu.cornell.kfs.concur.batch.businessobject;
+
+import java.sql.Date;
+
+import org.kuali.kfs.krad.bo.PersistableBusinessObjectBase;
+
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+public class ConcurRequestedCashAdvance extends PersistableBusinessObjectBase{
+    protected int concurRequestedCashAdvanceId;
```

Comment:
In that case, do you think INTEGER is large enough for what you need, or should it be BIGINT for this field instead?

Just for reference, Kuali BOs use Long and BIGINT for the version number field.